widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -16,6 +16,7 @@ internal class EntryWidgetController(private val queueMonitor: QueueMonitor) : E
     @SuppressLint("CheckResult")
     override fun setView(view: EntryWidgetContract.View) {
         this.view = view
+        // TODO: code below should be moved to a standalone component
```

Comment:
@DavDo I am thinking about it.
Have you added this TODO intentionally or by mistake, though?

Reply:
It came with a stash. I will remove it!